code/game/objects/items/weapons/blades.dm

```diff
@@ -48,13 +48,18 @@
 	desc = "A finely made Japanese sword, with a well sharpened blade. The blade has been filed to a molecular edge, and is extremely deadly. Commonly found in the hands of mercenaries and yakuza."
 	icon_state = "katana"
 	flags_atom = CONDUCT
-	force = 50
+	force = 35
 	throwforce = 10
+	attack_speed = 9
 	sharp = IS_SHARP_ITEM_BIG
 	edge = 1
 	w_class = 3
 	attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 
+/obj/item/weapon/katana/attack(mob/living/carbon/M as mob, mob/living/carbon/user as mob)
+	playsound(loc, 'sound/weapons/bladeslice.ogg', 25, 1)
```

Comment:
Remove this and make add `hitsound = 'sound/weapons/bladeslice.ogg'` to the object vars.
This is a copypasta of some patented idiot tier code. (If you want, apply this medicine to machetes too.)